src/main/java/net/minestom/server/command/ArgumentsSignature.java

```diff
@@ -0,0 +1,71 @@
+package net.minestom.server.command;
+
+import net.minestom.server.command.builder.arguments.Argument;
+import net.minestom.server.command.builder.arguments.minecraft.SignableArgument;
+import net.minestom.server.crypto.MessageSignature;
+import net.minestom.server.utils.binary.BinaryReader;
+import net.minestom.server.utils.binary.BinaryWriter;
+import net.minestom.server.utils.binary.Writeable;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public record ArgumentsSignature(long timestamp, long salt, Map<String, byte[]> signatures, boolean signedPreview)
+implements Writeable {
```

Comment:
Is this correct wrapping?

Reply:
Well `implements` was on the vertical guide line, should i keep it as a long line or wrap it in the middle of the args?

Reply:
I only ask because there's another instance in here where `implements` is wrapped and aligned with an 8 space indent, which means this is inconsistent. This is why I usually avoid wrapping on things like `implements`, though unfortunately, there isn't really much else you can wrap on here, except the parameters.